docs/planning/extensions.mdx

```diff
@@ -0,0 +1,64 @@
+# Extensions
```

Comment:
Idle thought: Would it make sense for this to be an `Advanced` topic, ie `Advanced - Incons`?